plex binding throwing exception when plex server goes offline

openhab.log:

```
2015-06-13 20:06:43.598 [ERROR] [.b.plex.internal.PlexConnector] - Error connecting to Plex
java.util.concurrent.ExecutionException: java.util.concurrent.TimeoutException: No response received after 10000
	at com.ning.http.client.providers.netty.NettyResponseFuture.get(NettyResponseFuture.java:265) ~[async-http-client-1.8.4.jar:na]
	at com.ning.http.client.providers.netty.NettyResponseFuture.get(NettyResponseFuture.java:222) ~[async-http-client-1.8.4.jar:na]
	at org.openhab.binding.plex.internal.PlexConnector.open(PlexConnector.java:157) [bundlefile:na]
	at org.openhab.binding.plex.internal.PlexConnector.connect(PlexConnector.java:357) [bundlefile:na]
	at org.openhab.binding.plex.internal.PlexConnector.access$4(PlexConnector.java:351) [bundlefile:na]
	at org.openhab.binding.plex.internal.PlexConnector$PlexWebSocketListener.onClose(PlexConnector.java:403) [bundlefile:na]
	at com.ning.http.client.providers.netty.NettyWebSocket.onClose(NettyWebSocket.java:237) [async-http-client-1.8.4.jar:na]
	at com.ning.http.client.providers.netty.NettyWebSocket.close(NettyWebSocket.java:128) [async-http-client-1.8.4.jar:na]
	at com.ning.http.client.providers.netty.NettyAsyncHttpProvider$WebSocketProtocol.onClose(NettyAsyncHttpProvider.java:2439) [async-http-client-1.8.4.jar:na]
	at com.ning.http.client.providers.netty.NettyAsyncHttpProvider.channelClosed(NettyAsyncHttpProvider.java:1469) [async-http-client-1.8.4.jar:na]
	at org.jboss.netty.channel.SimpleChannelUpstreamHandler.handleUpstream(SimpleChannelUpstreamHandler.java:88) [netty-3.9.0.Final.jar:na]
	at org.jboss.netty.channel.DefaultChannelPipeline.sendUpstream(DefaultChannelPipeline.java:564) [netty-3.9.0.Final.jar:na]
	at org.jboss.netty.channel.DefaultChannelPipeline$DefaultChannelHandlerContext.sendUpstream(DefaultChannelPipeline.java:791) [netty-3.9.0.Final.jar:na]
	at org.jboss.netty.handler.codec.replay.ReplayingDecoder.cleanup(ReplayingDecoder.java:570) [netty-3.9.0.Final.jar:na]
	at org.jboss.netty.handler.codec.frame.FrameDecoder.channelClosed(FrameDecoder.java:371) [netty-3.9.0.Final.jar:na]
	at org.jboss.netty.channel.SimpleChannelUpstreamHandler.handleUpstream(SimpleChannelUpstreamHandler.java:88) [netty-3.9.0.Final.jar:na]
	at org.jboss.netty.channel.DefaultChannelPipeline.sendUpstream(DefaultChannelPipeline.java:564) [netty-3.9.0.Final.jar:na]
	at org.jboss.netty.channel.DefaultChannelPipeline.sendUpstream(DefaultChannelPipeline.java:559) [netty-3.9.0.Final.jar:na]
	at org.jboss.netty.channel.Channels.fireChannelClosed(Channels.java:468) [netty-3.9.0.Final.jar:na]
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.close(AbstractNioWorker.java:375) [netty-3.9.0.Final.jar:na]
	at org.jboss.netty.channel.socket.nio.NioWorker.read(NioWorker.java:93) [netty-3.9.0.Final.jar:na]
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:108) [netty-3.9.0.Final.jar:na]
	at org.jboss.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318) [netty-3.9.0.Final.jar:na]
	at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:89) [netty-3.9.0.Final.jar:na]
	at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178) [netty-3.9.0.Final.jar:na]
	at org.jboss.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108) [netty-3.9.0.Final.jar:na]
	at org.jboss.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42) [netty-3.9.0.Final.jar:na]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145) [na:1.7.0_40]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615) [na:1.7.0_40]
	at java.lang.Thread.run(Thread.java:724) [na:1.7.0_40]
Caused by: java.util.concurrent.TimeoutException: No response received after 10000
	at com.ning.http.client.providers.netty.NettyResponseFuture.get(NettyResponseFuture.java:259) ~[async-http-client-1.8.4.jar:na]
	... 29 common frames omitted

```
and that happens every few seconds if the server is offline. It makes sense that the binding is not working but shouldn't it fail more gracefully?

--- a/bundles/binding/org.openhab.binding.plex/src/main/java/org/openhab/binding/plex/internal/PlexConnector.java
+++ b/bundles/binding/org.openhab.binding.plex/src/main/java/org/openhab/binding/plex/internal/PlexConnector.java
@@ -356,11 +356,11 @@ public class PlexConnector extends Thread {
 				Thread.sleep(delay);
 				open();
 			} catch (IOException e) {
-				logger.error("Error connecting to Plex", e);
+				logger.debug("Error connecting to Plex", e);
 			} catch (InterruptedException e) {
-				logger.error("Interrupted while reconnecting to Plex", e);
+				logger.debug("Interrupted while connecting to Plex", e);
 			} catch (ExecutionException e) {
-				logger.error("Error connecting to Plex", e);
+				logger.debug("Error connecting to Plex", e);
 			}
 			delay = RECONNECT_DELAY;
 		}
@@ -389,7 +389,7 @@ public class PlexConnector extends Thread {
 			} else if (e instanceof TimeoutException) {
 				logger.debug("[{}]: Websocket timeout error", connection.getHost());
 			} else {
-				logger.error("[{}]: Websocket error: {}", connection.getHost(), e.getMessage());
+				logger.debug("[{}]: Websocket error: {}", connection.getHost(), e.getMessage());
 			}
 		}
 		

